.github/workflows/publish.yml

```
@@ -0,0 +1,28 @@
+name: Upload Python Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --user --upgrade setuptools wheel
+    - name: Build
+      run: |
+        python setup.py sdist bdist_wheel
```

Calling setup.py directly is deprecated (long version: https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html)

It's recommended to use `build` instead (short version: https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html#summary)

```suggestion
    - name: Install dependencies
      run: |
        python -m pip install --upgrade build
    - name: Build
      run: |
        python -m build
```